lib/view_model/after_auth_view_models/chat_view_models/select_contact_view_model.dart

```diff
@@ -12,20 +9,35 @@ import 'package:talawa/view_model/base_view_model.dart';
 /// * `getCurrentOrgUsersList` : to get all users of current organization.
 class SelectContactViewModel extends BaseModel {
   late OrganizationService _organizationService;
+
+  /// orgMembersList is used to store all users of current organization.
   late List<User> orgMembersList = [];
 
-  // initialisation
+  /// This function initializes the [SelectContactViewModel] class.
+  ///
+  /// more_info_if_required
+  ///
+  /// **params**:
+  ///   None
+  ///
+  /// **returns**:
+  ///   None
   void initialise() {
     _organizationService = locator<OrganizationService>();
   }
 
-  /// This function is used to get all users list of an current organization.
+  /// function to get all users of current organization.
```

Comment:
Capitalize the first letter.

Reply:
Sure Sir I will fix this doc issue as well and try to open a PR asap.